CHANGELOG.md

```diff
@@ -56,6 +56,7 @@
 
 ## stream-chat-android-ui-components
 ### 🐞 Fixed
+- Fixed crash in `MessageListView` when posting a message to en empty channel. [#5332](https://github.com/GetStream/stream-chat-android/pull/5332)
```

Comment:
Typo?

```suggestion
- Fixed crash in `MessageListView` when posting a message to an empty channel. [#5332](https://github.com/GetStream/stream-chat-android/pull/5332)
```

Reply:
Yeah, a typo :)

@skydoves thanks for noticing this!

@JcMinarro thanks for fixing this!